code/modules/sprite_accessories/accessory_human.dm

```diff
@@ -82,17 +82,3 @@
 	icon_state = "horns_frills_long"
 	name = "Horn Biomods (Long Frills)"
 
-
-/datum/sprite_accessory/marking/human/horns/frills_short
-	icon_state = "horns_frills_short"
-	name = "Horn Biomods (Short Frills)"
```

Comment:
Frills has been deleted. This should be restored.

Reply:
This is getting embarrassing for me. Will do

Reply:
Nothing to fret over. Happens to all of us :+1: